source/connect/connections.txt

```diff
@@ -0,0 +1,81 @@
+.. _compass-sidebar:
+
+===============
+Compass Sidebar
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   .. include:: /includes/multiple-connections.rst
+
+
+The Compass sidebar on the left of the Compass window
+contains the following options:
+
+- :icon-lg:`Plus` icon to add new connections
+- :icon-fa5:`ellipsis-h` icon to import and export data
```

Comment:
I would specifically say "to import and export connections" to be super clear (because Compass has different data import/export functionality)

Reply:
Done!